### ISSUE TITLE
xf:message dismissed with the Esc key disables further messages

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#2333]
- trigger `<xf:message>`
- Esc
- trigger `<xf:message>` again
- doesn't show

**Comment by NightingaleCrypter** [comment]
Issue might be deprecated due to moving to browser native window alert on usage of xf:message.

**Comment by avernet** [comment]
You're right; thanks for the comment. `xf:message` is now using `window.alert()`. This was done as a part of #2835 for 2024.1. Closing.
